src/components/Send.tsx

```diff
@@ -1,4 +1,151 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter, RouteComponentProps } from 'react-router';
+import { Form, FormGroup, Input, Label, Button, Row, Col, Alert } from 'reactstrap';
+import { getWallets, selectActiveWallet } from '../store/wallet';
+import { AppState } from '../store';
+import api, { CreatePSBTOutput } from '../api';
+import { Wallet } from '../types';
+import './Send.css'
 
-// TODO: Send page
-export default (() => <>I'm the Send page</>) as React.SFC;
+interface DispatchProps {
+  getWallets: typeof getWallets;
+}
+
+interface ConnectProps {
+  // FIXME: typescript makes me accept possibility of nullness
+  // How can we just demand that this not be null for the component to even load?
+  activeWallet: Wallet | null;
+}
+
+type Props = DispatchProps & ConnectProps & RouteComponentProps;
+
+interface StateProps {
+  activeWallet: Wallet | null;
+}
+
+interface LocalState {
+  outputs: CreatePSBTOutput[];
+  isSubmitting: boolean;
+  error: Error | null;
+}
+
+class Create extends React.Component<Props, LocalState> {
+  state: LocalState = {
+    outputs: [{'address': undefined, 'btc': undefined}],
+    isSubmitting: false,
+    error: null,
+  };
+
+  private handleChangeRecipient = (ev: React.ChangeEvent<HTMLInputElement>, index: number) => {
+    let { outputs } = this.state;
+    (outputs[index] as any)[ev.currentTarget.name] = ev.currentTarget.value;
+    this.setState({ outputs });
+  };
+
+  private handleChangeAmount = (ev: React.ChangeEvent<HTMLInputElement>, index: number) => {
+    let { outputs } = this.state;
+    (outputs[index] as any)[ev.currentTarget.name] = Number(ev.currentTarget.value);
+    this.setState({ outputs });
+  };
+
+  private handleAddOutput = (ev: React.FormEvent<HTMLInputElement>) => {
+    let { outputs } = this.state;
+    outputs.push({ address: undefined, btc: undefined });
+    this.setState({ [ev.currentTarget.name]: ev.currentTarget.value } as any);
+  };
+
+  private handleRemoveOutput = (index: number) => {
+    let state = this.state;
+    state.outputs.splice(index, 1);
+    this.setState(state);
+  };
+
+  private handleSubmit = async (ev: React.FormEvent<HTMLFormElement>) => {
+    ev.preventDefault();
+    try {
+      this.setState({ isSubmitting: true });
+      const { outputs } = this.state;
+      const { activeWallet } = this.props;
+      await api.createPSBT({
+        wallet_name: activeWallet ? activeWallet.name : '',  // FIXME
```

Comment:
This is outdated. With `bootstrap` we should have guarantee that `activeWallet` exists ...